Fix incorrect prefix in Arm NEON shift right narrow intrinsic

Fixes build error caused by 'simde_vshrn_n_u16()' being seen as
undeclared in Clang 15, where the -Wimplicit-function-declaration
warning diagnostic defaults to an error in C99 and later.
---
 kitty/simd-string-impl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kitty/simd-string-impl.h b/kitty/simd-string-impl.h
index 3bbf95a15..6f9473a38 100644
--- kitty/simd-string-impl.h
+++ kitty/simd-string-impl.h
@@ -241,7 +241,7 @@ static inline integer_t shuffle_impl256(const integer_t value, const integer_t s
 
 static inline uint64_t
 movemask_arm128(const simde__m128i vec) {
-    simde_uint8x8_t res = simde_vshrn_n_u16(simde_vreinterpretq_u16_u8((simde_uint8x16_t)vec), 4);
+    simde_uint8x8_t res = vshrn_n_u16(simde_vreinterpretq_u16_u8((simde_uint8x16_t)vec), 4);
     return simde_vget_lane_u64(simde_vreinterpret_u64_u8(res), 0);
 }
 
-- 
2.44.0

